complex/src/lib.rs

```diff
@@ -178,7 +180,7 @@ impl<T: Clone + Float> Complex<T> {
         let (r, theta) = self.to_polar();
         Complex::from_polar(&(r.sqrt()), &(theta/two))
     }
-    
+
```

Comment:
Please don't include unrelated changes like this.  I won't ask you to revert these this time, because they're few enough that I can look past them, but generally it makes things harder to review.